### DISCUSSION TITLE
i_did+itwrongdidnt_i

### DISCUSSION THREAD

**Discussion by smiletsretarded123** [JordanSantiagoYT/FNF-JS-Engine#623] [Q&A] [answered]
BRUVVVV

basically  I put the song in just like psych engine and its playing some weird corny robotic song (test)

**Comment by JordanSantiagoYT** [comment]
you did

the song has to have an Inst and (if you have one) Voices file.

**Comment by moxie-coder** [comment]
@JordanSantiagoYT go back in time and prevent me from being born

**Comment by smiletsretarded123** [comment]
i just did this 
![Screenshot 2024-09-29 103149](https://github.com/user-attachments/assets/9fcac846-4512-4bcc-ac02-3d6237ad3e84)
![image](https://github.com/user-attachments/assets/a21f9ed0-d2ee-4642-b083-b89ccb83e18a)
![image](https://github.com/user-attachments/assets/7161f105-4823-484f-b218-b05fd29af5dd)
![image](https://github.com/user-attachments/assets/504dc676-e27c-40e6-9dbb-ef9d79a216dc)

**Comment by JordanSantiagoYT** [comment]
.........
you werent supposed to put them in assets

**Reply by smiletsretarded123** [reply]
in mods?

**Comment by smiletsretarded123** [comment]
OH MY GOSH I LITTEERALLY JSUT REALIZED i looked in assets instead of mods thanks for the reminder!!!

**Comment by smiletsretarded123** [comment]
still is this correct
![image](https://github.com/user-attachments/assets/02e4feb2-55f7-4a67-8bf5-cc0b9772f615)
![image](https://github.com/user-attachments/assets/23dc75bb-220e-4017-9e9d-093ac6a412e0)
![image](https://github.com/user-attachments/assets/ceacebc1-f552-424c-b3e6-dbc8a8ddb064)
![image](https://github.com/user-attachments/assets/17634279-ebb1-4954-8eff-3cbfd04f3411)
![image](https://github.com/user-attachments/assets/f1220cda-11e9-4724-a3c8-6f996f85fa1d)

**Reply by moxie-coder** [reply]
> still is this correct ![image](https://private-user-images.githubusercontent.com/181041745/371880151-02e4feb2-55f7-4a67-8bf5-cc0b9772f615.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzUyMDQsIm5iZiI6MTcyNzYzNDkwNCwicGF0aCI6Ii8xODEwNDE3NDUvMzcxODgwMTUxLTAyZTRmZWIyLTU1ZjctNGE2Ny04YmY1LWNjMGI5NzcyZjYxNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyOVQxODM1MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NjY3NTcwMzBhNTQ0ZDg4YzRiYjExMDFlODdlOWUyMDRmN2E5MjIxMWQ1ZDcwZjA4ZTQ3NGUxOTgxMTU2YzIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.3H2YjpwIFx3xjgW1RK5tWWmQFwturBiY9W_701-YzEw) ![image](https://private-user-images.githubusercontent.com/181041745/371880158-23dc75bb-220e-4017-9e9d-093ac6a412e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzUyMDQsIm5iZiI6MTcyNzYzNDkwNCwicGF0aCI6Ii8xODEwNDE3NDUvMzcxODgwMTU4LTIzZGM3NWJiLTIyMGUtNDAxNy05ZTlkLTA5M2FjNmE0MTJlMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyOVQxODM1MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NDlhOTAwYjhkZDVmMTlkMmI1NWZlMDYwZmI1MDg5ZDI2NWRkOTRiNTM4MWI0YmVkZTU5YjQ5YjVlM2E4M2IxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZF1O9b4xoVs82lmlLGY0GXp12ZnV9NWXKX0TmhOaJVs) ![image](https://private-user-images.githubusercontent.com/181041745/371880174-ceacebc1-f552-424c-b3e6-dbc8a8ddb064.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzUyMDQsIm5iZiI6MTcyNzYzNDkwNCwicGF0aCI6Ii8xODEwNDE3NDUvMzcxODgwMTc0LWNlYWNlYmMxLWY1NTItNDI0Yy1iM2U2LWRiYzhhOGRkYjA2NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyOVQxODM1MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNjc3OGRkMGUxM2M3ODMwMGUzZmZjYzEzMjkxN2Q1ZWZhOWRlYWEwZTQ4MDAyYTgzNWYyMTI4NzJiZTExMzZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.1B8cuxyOeYjIo4ApC7pcjTn1X1zX52kz3Tvw7COnOf4) ![image](https://private-user-images.githubusercontent.com/181041745/371880187-17634279-ebb1-4954-8eff-3cbfd04f3411.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzUyMDQsIm5iZiI6MTcyNzYzNDkwNCwicGF0aCI6Ii8xODEwNDE3NDUvMzcxODgwMTg3LTE3NjM0Mjc5LWViYjEtNDk1NC04ZWZmLTNjYmZkMDRmMzQxMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyOVQxODM1MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZjBkYTVjNTc2MDUwODk5MWNiMjUwMDRhYWM1ZWU5NmUwMjMwYzc5ZDZhZDBlNDVjOTJmZmI1MDQyYTZkYzM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZaenKGLU93I5Wt5tigY8RfsEDu6vIivfARVq-Y80GmA) ![image](https://private-user-images.githubusercontent.com/181041745/371880198-f1220cda-11e9-4724-a3c8-6f996f85fa1d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzUyMDQsIm5iZiI6MTcyNzYzNDkwNCwicGF0aCI6Ii8xODEwNDE3NDUvMzcxODgwMTk4LWYxMjIwY2RhLTExZTktNDcyNC1hM2M4LTZmOTk2Zjg1ZmExZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyOVQxODM1MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYTc4NTNiZWI0ODg3MmFkYzczOTc2ZDAxN2E5ZjI5MDNmNjQyZDA1M2JiMDYxYWVkNWU5ODg4ZDdiZDE5YjM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.isa2DQjwXlouge49fuwj9DEhGMUSsV7H8X5uKUTkd2E)

yes

**Comment by moxie-coder** [comment, marked as answer]
you put them in mods, not assets
